templates/translations.html.twig

```diff
@@ -154,6 +154,7 @@
 {{ 'profile.saml.attributes.serialNumber'|trans({}, 'saml') }}
 {{ 'profile.saml.attributes.singleLevelQuality'|trans({}, 'saml') }}
 {{ 'profile.saml.attributes.surf-crm-id'|trans({}, 'saml') }}
+{{ 'profile.saml.attributes.surf-autorisaties'|trans({}, 'saml') }}
```

Comment:
Smell: `autorisaties` is Dutch?

Reply:
Merge from @thijskh , did not touch this